cnf-tests/testsuites/e2esuite/dpdk/dpdk.go

```diff
@@ -958,43 +1058,15 @@ func createDPDKWorkload(nodeSelector map[string]string, dpdkResourceName, testpm
 		Command: []string{
 			"/bin/bash",
 			"-c",
-			fmt.Sprintf(`set -ex
-export CPU=$(cat /sys/fs/cgroup/cpuset/cpuset.cpus)
-echo ${CPU}
-echo ${PCIDEVICE_OPENSHIFT_IO_%s}
-
-cat <<EOF >test.sh
-spawn %s
-set timeout 10000
-expect "testpmd>"
-send -- "port stop 0\r"
-expect "testpmd>"
-send -- "port detach 0\r"
-expect "testpmd>"
-send -- "port attach ${PCIDEVICE_OPENSHIFT_IO_%s}\r"
-expect "testpmd>"
-send -- "port start 0\r"
-expect "testpmd>"
-send -- "start\r"
-expect "testpmd>"
-sleep %d
-send -- "stop\r"
-expect "testpmd>"
-send -- "quit\r"
-expect eof
-EOF
-
-expect -f test.sh
-
-sleep INF
-`, dpdkResourceName, testpmdCommand, dpdkResourceName, runningTime)},
+			command,
+		},
 		SecurityContext: &corev1.SecurityContext{
 			RunAsUser: pointer.Int64Ptr(0),
 			Capabilities: &corev1.Capabilities{
 				// Enable NET_RAW is required by mellanox nics as they are using the netdevice driver
 				// NET_RAW was removed from the default capabilities
 				// https://access.redhat.com/security/cve/cve-2020-14386
-				Add: []corev1.Capability{"IPC_LOCK", "SYS_RESOURCE", "NET_RAW"},
+				Add: []corev1.Capability{"IPC_LOCK", "SYS_RESOURCE", "NET_RAW", "NET_ADMIN", "MKNOD"},
```

Comment:
please add them only for the vhost-net test. 
we want to check that for regular dpdk NET_ADMIN is not needed and want to say as low as possible on capabilities